File: voip/discordSIP/app/sip_audio.py
import pjsua2 as pj

def createWavPlayer() -> pj.AudioMediaPlayer:
    player = pj.AudioMediaPlayer()
    try:
        player.createPlayer("/app/test.wav")
    except Exception as e:
        print(e)
    return player

def getSpeaker() -> pj.AudioMedia:
    return pj.Endpoint.instance().audDevManager().getPlaybackDevMedia()

File: voip/discordSIP/appJDA/app/src/main/java/discord2voip/pjsua2/SipAudio.java
package discord2voip.pjsua2;

import org.pjsip.pjsua2.AudioMedia;
import org.pjsip.pjsua2.AudioMediaPlayer;
import org.pjsip.pjsua2.Endpoint;

public class SipAudio {

    public AudioMediaPlayer createWavPlayer() {
        AudioMediaPlayer player = new AudioMediaPlayer();
        try {
            player.createPlayer("/app/test.wav");
        } catch (Exception e) {
            e.printStackTrace();
        }
        return player;
    }

    public AudioMedia getSpeaker() {
        try {
            return Endpoint.instance().audDevManager().getPlaybackDevMedia();
        } catch (Exception e) {
            e.printStackTrace();
            return null;
        }
    }
}
